android/app/src/main/java/org/pathcheck/covidsafepaths/MainApplication.java

```diff
@@ -85,17 +83,6 @@ private static void initializeFlipper(Context context) {
     }
   }
 
-  private void initializeGeolocationTransformer() {
```

Comment:
I think removing this restores default storage behaviour, which will leave all the unencrypted data in the underlying `react-native-background-geolocation` storage (sqlite)..


We probably still want this override to return `null`.

@AdamLeonSmith

Reply:
If it "transforms" the location to `null`, it won't get emitted on `BackgroundGeolocation.on('location'...)`:

https://github.com/mauron85/background-geolocation-android/blob/2693e07587eea08746004bd3e72dde276b04b4d2/src/main/java/com/marianhello/bgloc/service/LocationServiceImpl.java#L534

Reply:
But you're right, `react-native-background-geolocation` will persist it in its own storage now, which is also not what we want...